docs/varnish/vcl/varnish4.vcl

```diff
@@ -167,6 +163,35 @@ sub ez_purge {
     }
 }
 
+sub ez_purge_acl {
+    if  (!client.ip ~ invalidators) {
+        return (synth(405, "Method not allowed"));
+    }
+}
+
+## Alternative 1: ez_purge_acl for purging by Token (Varnish >= 5.1, std.getenv support)
+#sub ez_purge_acl {
+#    if (req.http.x-purge-token) {
+#        if (req.http.x-purge-token != std.getenv("ACL_INVALIDATE_TOKEN")) {
+#            return (synth(405, "Method not allowed"));
+#        }
+#    } else if  (!client.ip ~ invalidators) {
+#        return (synth(405, "Method not allowed"));
+#    }
+#}
+
+## Alternative 2: ez_purge_acl for purging by Token (Varnish <= 5.1, token value in vcl)
+#sub ez_purge_acl {
+#    if (req.http.x-purge-token) {
+#        # change the token "_YOUR_TOKEN_HERE_" before enabling it.
+#        if (req.http.x-purge-token != "_YOUR_TOKEN_HERE_") {
+#            return (synth(405, "Method not allowed"));
+#        }
+#    } else if  (!client.ip ~ invalidators) {
+#        return (synth(405, "Method not allowed"));
+#    }
+#}
```

Comment:
In retrospect this could have just been:
```
sub ez_purge_acl {
//    if (req.http.x-purge-token) {
//        #  Won't work on Varnish 4.1, if needed you can hardcode a secret token here instead of std.getenv() usage
//        if (req.http.x-purge-token != std.getenv("ACL_INVALIDATE_TOKEN")) {
//            return (synth(405, "Method not allowed"));
//        }
//    } else if  (!client.ip ~ invalidators) {
    if  (!client.ip ~ invalidators) {
        return (synth(405, "Method not allowed"));
    }
}
```
With corresponding simplification in parameters.vcl.

Reply:
Done